openqa-label-known-issues

```diff
@@ -35,7 +35,8 @@ echoerr() { echo "$@" >&2; }
 
 handle_unreachable() {
     local testurl=$1
-    local out=$2
+    local out
+    out=$(cat "$2")
```

Comment:
The content of the variable `out` is supposed to be a filename.
Now it contains the content of the autoinst-log.txt.
That will probably fail if it contains a slash or anything that cannot be part of a filename, and it's a mess anyway.
Also the file should be created via `mktemp`, as we do for the other file.

still, that's not enough, because we are calling
```
f [[ $(date -uIs -d '-14days') > $(grep timeago "$out" | hxselect -s '\n' -c '.timeago::attr(title)') ]]; then
```
later in line 144 (that's the part you extracted from `handle_unreachable` in one of the previous PRs), and that expects the HTML content in the `$out` file, not the autoinst log.
So the filename / variable has to be available at that point.
There should be two files, and two distinct variable names.